### ISSUE TITLE
Readme errors

### ISSUE THREAD

**Issue by stefandevo** [jonataslaw/get#144]
Hi I found some errors in the readme:

```
GetMaterialApp(
      enableLog: true,
      defaultTransition: Transitions.fade,
      defaultOpaqueRoute: Get.isOpaqueRouteDefault,
      defaultPopGesture: Get.isPopGestureEnable,
      defaultDurationTransition: Get.defaultDurationTransition,
      defaultGlobalState: Get.defaultGlobalState,
    );
```

defaultOpaqueRoute, defaultPopGesture, defaultDurationTransition are *not* available (anymore)

**Comment by jonataslaw** [comment]
> Hi I found some errors in the readme:
> 
> ```
> GetMaterialApp(
>       enableLog: true,
>       defaultTransition: Transitions.fade,
>       defaultOpaqueRoute: Get.isOpaqueRouteDefault,
>       defaultPopGesture: Get.isPopGestureEnable,
>       defaultDurationTransition: Get.defaultDurationTransition,
>       defaultGlobalState: Get.defaultGlobalState,
>     );
> ```
> 
> defaultOpaqueRoute, defaultPopGesture, defaultDurationTransition are _not_ available (anymore)

Thanks for letting me know about this, I will fix it in the next update.

**Comment by jonataslaw** [comment]
This was a typo, the api is active, but with another name.
I'll change the readme to:

```
GetMaterialApp(    
    enableLog: true,
    defaultTransition: Transition.fade,
    opaqueRoute: Get.isOpaqueRouteDefault,
    popGesture: Get.isPopGestureEnable,
    transitionDuration: Get.defaultDurationTransition,
    defaultGlobalState: Get.defaultGlobalState,
```

**Comment by jonataslaw** [comment]
Fix on 2.7.0
